sdcm/tester.py

```diff
@@ -1654,17 +1654,19 @@ def run_stress(self, stress_cmd, duration=None):
     # pylint: disable=too-many-arguments,too-many-return-statements
     def run_stress_thread(self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='',
                           round_robin=False, stats_aggregate_cmds=True, keyspace_name=None, use_single_loader=False,
-                          stop_test_on_failure=True):
+                          stop_test_on_failure=True, sla_role=None):
         # We want to prevent situation when stress command starts on not ready cluster (no all nodes are UP).
         # It may cause to stress failure and as result the test will be stopped and failed
         self.db_cluster.wait_for_nodes_up_and_normal(iterations=0, timeout=MAX_TIME_WAIT_FOR_ALL_NODES_UP)
 
         params = dict(stress_cmd=stress_cmd, duration=duration, stress_num=stress_num, keyspace_num=keyspace_num,
                       keyspace_name=keyspace_name, profile=profile, prefix=prefix, round_robin=round_robin,
-                      stats_aggregate_cmds=stats_aggregate_cmds, use_single_loader=use_single_loader)
+                      stats_aggregate_cmds=stats_aggregate_cmds, use_single_loader=use_single_loader,
+                      sla_role=sla_role)
 
         if 'cassandra-stress' in stress_cmd:  # cs cmdline might started with JVM_OPTION
             params['stop_test_on_failure'] = stop_test_on_failure
+            self.log.info("run_stress_thread.params: %s", params)
```

Comment:
Remember to remove info that was added for debug

Reply:
removed